source/common/modules/markdown-editor/plugins/render-codeblock-button.ts

```diff
@@ -41,6 +41,7 @@ const clipboard = window.clipboard
       codeblock = codeblock + cm.getLine(++j) + '\n'
       incodeblock = true
     } else if (codeBlockRE.test(line) && incodeblock) {
+      // Update the codes in code-block
```

Comment:
Sorry in advance for nit-picking with the comments. But we should keep in mind that Zettlr is an open-source project. Therefore, we should make the comments as descriptive as possible. So anyone visiting this line in the future should immediately understand its purpose. In this case, something like "_Remove newline character after the last code line_" tells us directly why this line is here.

You could place it below the current "Update the codes in code-block" comment.

Reply:
Yes, that looks better, thanks!